lib/mayaUsd/ufe/UsdTransform3dMayaXformStack.cpp

```diff
@@ -174,23 +174,23 @@ class UsdTRSUndoableCmdBase : public Ufe::SetVector3dUndoableCommand
 public:
     struct State
     {
-        virtual const char* name() const = 0;
-        virtual void        handleUndo(UsdTRSUndoableCmdBase*)
+        virtual void handleUndo(UsdTRSUndoableCmdBase*)
```

Comment:
Minor cleanup done to the UsdTransform3dMayaXformStack.cpp state machine, when I was hopeful I could share this code with the matrix op and common API Transform3d interfaces.  This ended up being too much work, but I kept the improvements.